test/expect-tests/dune_rpc/dune_rpc_tests.ml

```diff
@@ -57,10 +57,10 @@ let setup_client_server () =
 let test ?(private_menu = []) ?no_real_methods ~client ~handler ~init () =
```

Comment:
Wouldn't `?no_real_methdos` be better as `?(no_real_methods = false)`?

Reply:
Yes indeed.